user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/General/Stepper.md

```diff
@@ -21,6 +21,11 @@ To configure the component:
 
    - When a DOM instance is applied, this will be used to display the current state, preceded by the states the instance has traversed until its current state, and followed by the future states it will go through if it follows the normal flow.
 
+     > [!NOTE]
+     > Not all DOM instances keep a history of the states they traversed, this behaviour is decided by the module. When an instance has no history, the path from the first state to the current state is calculated in a similar way as the happy path after the current state. Concretely, this means you'll see the states the instance traverse through to get from the first possible state to the current state if it were to follow the normal flow. 
```

Comment:
reading this, it seems to me that for some instances in a module there is history and for others not, which is not the case. Maybe explain that it is a setting on the module that can be enabled/disabled (with a link to those docs)